happi/loader.py

```diff
@@ -177,6 +177,14 @@ def create_arg(arg):
     args = [create_arg(arg) for arg in item.args]
     kwargs = dict((key, create_arg(val))
                   for key, val in item.kwargs.items())
+    # maybe filter out null kwargs
+    if not item._info_attrs['kwargs'].include_default_as_kwarg:
+        new_kwargs = {}
+        for k, v in kwargs.items():
+            einfo = item._info_attrs.get(k, False)
+            if not einfo or not einfo.default == v:
+                new_kwargs[k] = v
```

Comment:
I always find single-letter variable name loops hard to read. This is maybe a personal preference of mine rather than a complaint here. I probably would have used `entry`, `value` and agonized over whether that was the clearest word choice.

Reply:
I'm not opposed to more full variable names, I suppose I use these single letter names if I really don't intend for them to be used elsewhere.